eventRequirement.js

```diff
@@ -634,6 +652,10 @@ OperandVariable.prototype.setPointers = function (entitiesArr) {
         this.operandValueOrObject().left.setPointers(entitiesArr);
         this.operandValueOrObject().right.setPointers(entitiesArr);
     }
+    if (this.operandType() == "arrayvalue") {
```

Comment:
please reuse the arithmetic case. For example:
if (this.operandType() == "arithmetic" || this.operandType() == "arrayvalue")

Reply:
Done